pkgs/applications/misc/bellepoule/default.nix

```diff
@@ -0,0 +1,91 @@
+{ lib
+, stdenv
+, fetchgit
+, goocanvas
+, pkg-config
+, gtk2
+, libxml2
+, curl
+, libmicrohttpd
+, libzip
+, libusb1
+, qrencode
+, json-glib
+, php82
+, libwebsockets
+, openssl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "bellepoule";
+  version = "5.4";
+
+  src = (fetchgit {
+    url = "https://git.launchpad.net/bellepoule";
+    rev = version;
+    hash = "sha256-PFSbmdzqfB2rNtLGcQWfw73Ja9CC4z4Z588oKj4gRSE=";
+  }).overrideAttrs (_: {
+    GIT_CONFIG_COUNT = 1;
+    GIT_CONFIG_KEY_0 = "url.https://github.com/.insteadOf";
+    GIT_CONFIG_VALUE_0 = "git@github.com:";
+  });
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    gtk2
+    libxml2
+    curl
+    libmicrohttpd
+    goocanvas
+    qrencode
+    openssl
+    json-glib
+    libzip
+    libusb1
+    libwebsockets
+  ];
+
+  patches = [
+    # Add placeholder Twitter / Facebook credentials.
+    ./empty-advertiser_ids.patch
+    # webkit-1.0 is optional and can be disabled in the MakeFile (only used for Twitter / Facebook authorization).
+    ./remove-webkit.patch
+  ];
+
+  # Use system php
+  postPatch = ''
+    sed -i -e "s#php7.0#${php82}/bin/php#g" ./sources/common/network/web_server.cpp
+  '';
+
+  # Fix homeshelter (MakeFile builds in $HOME)
+  # Package resources & sources for release
+  buildPhase = ''
```

Comment:
When specifying phases, please keep the phase hooks at start/end
https://nix.dev/tutorials/learning-journey/packaging-existing-software#phases-and-hooks